runtime/src/ancient_append_vecs.rs

```diff
@@ -628,20 +630,20 @@ pub mod tests {
             .map(|store| db.get_unique_accounts_from_storage(store))
             .collect::<Vec<_>>();
         let storage = storages.first().unwrap().clone();
-        let pk2 = solana_sdk::pubkey::new_rand();
+        let pk_1_ref = solana_sdk::pubkey::new_rand();
```

Comment:
why `_ref`, this is an owned `Pubkey`?

Reply:
Oh...I see this pubkey will have 1 reference.
What would you think about something like `pk_with_1_ref` (I know it's very explicit 😬 )